### PR TITLE
Support redesign of LVZ website

### LINKED ISSUE #100: Crawling fails after the redesign of LVZ website (since 02.03.2022)
Since 02 March 2022 all articles under http://www.lvz.de/Leipzig/Polizeiticker/Polizeiticker-Leipzig/ are dead links (not found).

The new starting page is https://www.lvz.de/themen/leipzig-polizei/ and the site structure and pagination has totally changed.

### PR DESCRIPTION
**Pull request by inpercima**
* fix #100

### CODE REVIEW

**Comment by inpercima** [comment]
@sepe81 This is just a beginning to get the data collected.

Details:
* title: can be found
* article: currently a problem, I do not have a paid account, I will check later
* teaser: can be found
* copyright: no idea to search it
* date published: can be found, now this value is stored in time-tag in the attribute "datetime"

File: src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java [hunk 1]
```diff
@@ -136,19 +135,15 @@ private static void extractCopyright(final Document doc, final PoliceTicker dm)
      * Try to extract the publishing date from a script block.
      *
      * @param doc Document
-     * @param dm PoliceTicker
+     * @param dm  PoliceTicker
```
Comment (sepe81) [inline src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java:138]: Revert line?

**Review by sepe81 — COMMENTED** [review]
LGTM, good progress

File: src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java (hunk 1 above)
Comment (inpercima) [reply src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java:138]: Yes I will do this. It's a formatter, normally for ts, to set the doc in columns for better reading.

Comment (inpercima) [reply src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java]: Done.

**Comment by inpercima** [comment]
Following data now can be crawled:

* title - same intention as before
* teaser - same intention as before
* article - currently some adds includes (will be removed), headlines are not filtered
  if the article is a paid one, the article is **now empty**
* copyright - now "© Verlagsgesellschaft Madsack GmbH & Co. KG" instead of "© Leipziger Verlags- und Druckereigesellschaft mbH & Co. KG"
* date published - now over a html-time-tag (datetime attribute)

What is do to?
* pagination needs to be clarified
* remove adds in article
* fill article with teaser if empty?

File: src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java [hunk 2]
```diff
@@ -181,11 +176,8 @@ static Date extractDate(final String date) {
 
     private static void extractArticle(final Document doc, final PoliceTicker dm) {
         final var content = "articlecontent";
-        var cssQuery = ".pdb-article-body > .pdb-richtext-field > p";
```
Comment (inpercima) [inline src/main/java/de/codefor/le/crawler/LvzPoliceTickerDetailViewCrawler.java:184]: This block can be removed. If a article is not a paid one, teaser and article can be simple found. If the article is a paid one, the teaser will be found too but the article is not visible, so articlecontent will be emty.

**Comment by inpercima** [comment]
While analysing the "load more" button I found out, that this is a javascript AJAX request, which can't be loaded by JSOUP. There is now a API we can use. On click on the button following API call is made:

https://www.lvz.de/pf/api/v3/content/fetch/Themenfeeds?query={"contentTypes":"story","offset":"25","size":10,"sortfield":"display_date","tags":"leipzig-polizei"}d=98_website=leipziger-volkszeitung

I do not know what the parameters "d" and "_website" stand for but if we remove these and replace "offset 25" with "offset 0" and "size 10" with "size 100" all last 100 stories are given with many more information.

https://www.lvz.de/pf/api/v3/content/fetch/Themenfeeds?query={"contentTypes":"story","**offset":"0"**,"size":100,"sortfield":"display_date","tags":"leipzig-polizei"}

All DOM analysis are no more necessary.

**Comment by inpercima** [comment]
In consultation with @sepe81, we don't want to use the API for the moment, but try to load the other articles with other technology. Here I use Selenium to click the button "Mehr anzeigen" and then pass the result to JSOUP. This works.

**Comment by inpercima** [comment]
Oh some tests need to be fixed.

**Comment by inpercima** [comment]
> Oh some tests need to be fixed.

Tests now fixed.

**Comment by inpercima** [comment]
@sepe81 What did you think? Is this PR mergeable?

**Review by sepe81 — APPROVED** [review]
Ready for merge, thanks to @inpercima for the impressive work!

**Pull request merged**